tests/form/robot/ValidationTextBox.html

```diff
@@ -495,13 +495,39 @@
 							doh.robot.typeKeys('123', 1000, 600);
 							doh.robot.sequence(d.getTestCallback(dojo.hitch(this, function(){
 								doh.is('123', this.textbox.focusNode.value, "focusNode.value");
-								doh.is(undefined, this.textbox.get('value'), "get('value')");
+								doh.is(123, this.textbox.get('value'), "get('value')");
 								doh.f(this.textbox.isValid(), "!isValid()");
```

Comment:
I tried them together at first but had and issue.  I copied another onblur
test to get them working.  I will attempt to merge them again.

On Fri, May 8, 2015 at 5:06 PM, Bill Keese <notifications@github.com> wrote:

> In tests/form/robot/ValidationTextBox.html
> <https://github.com/dojo/dijit/pull/93#discussion_r29980787>:
>
> > @@ -495,13 +495,39 @@
> >  							doh.robot.typeKeys('123', 1000, 600);
> >  							doh.robot.sequence(d.getTestCallback(dojo.hitch(this, function(){
> >  								doh.is('123', this.textbox.focusNode.value, "focusNode.value");
> > -								doh.is(undefined, this.textbox.get('value'), "get('value')");
> > +								doh.is(123, this.textbox.get('value'), "get('value')");
> >  								doh.f(this.textbox.isValid(), "!isValid()");
>
> Hmm, maybe it is correct for isValid() to return until the blur.
> Relatedly, presumably before the blur, ValidationTextBox#state is
> "Incomplete" rather than "Normal"? But why does get("value") return a
> valid value at that point? That's the part that seems inconsistent.
>
> On a more minor point, can't the "missing required decimal" and "missing
> required decimal, format on blur" test be combined into one test? Type
> "123", check conditions, then blur, then check conditions again.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dojo/dijit/pull/93/files#r29980787>.
>